tests/features/communities/eif/vocabulary.feature

```diff
@@ -45,3 +45,34 @@ Feature:
     When I go to "/solutions"
     Then I should see the "Some EIF solution" tile
     But I should not see the text "EIF reference"
+
+  @terms
+  Scenario: Solutions referencing an EIF term should appear in the corresponding page.
+    Given collection:
+      | title | EIF Toolbox |
+      | state | validated   |
+    And solutions:
+      | title    | state     | policy domain | collection  | eif reference                                                                             |
+      | Balker   | validated |               | EIF Toolbox | Underlying Principle 1: subsidiarity and proportionality, Underlying Principle 2: openess |
```

Comment:
There is a typo in this taxonomy term name, `openess` should be spelled `openness`. Do we have any control over the terms?

Reply:
Checking with Sander.

Reply:
Sander will report it.